File: Selenium/Project/Activity1.java
package JOBBOARD;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class Activity1 {

    WebDriver driver;
    @BeforeClass
    public void Beforeclass() {
    System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
    WebDriverManager.firefoxdriver().setup();
    driver = new FirefoxDriver();
    driver.get("https://alchemy.hguy.co/jobs");

}
    @Test
    public void getTitle()
{
    String title= driver.getTitle();
    System.out.println("The Title Is " + title);
    Assert.assertEquals("Alchemy Jobs – Job Board Application",title);
    String til="Alchemy Jobs";
    if (til == title)
    {
    driver.close(); }

}

}

File: Selenium/Project/Activity1.py
# Import webdriver from selenium
import pytest
import time

from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())
# Start the Driver
with webdriver.Firefox(service=service) as driver:
    # Open the browser to the URL
    driver.get("https://alchemy.hguy.co/jobs")
    print("Page Title is ", driver.title)
    time.sleep(8)

    title=driver.title
    print("The itle Is " + title)
    til="Alchemy Jobs"
    if til == title:
     driver.close()



